packages/core/config/storybook/preview.js

```diff
@@ -14,3 +14,11 @@ require('~__REDWOOD__USER_WEB_DEFAULT_CSS')
 addDecorator((storyFn, { id }) => {
   return React.createElement(StorybookProvider, { storyFn, id })
 })
+
+try {
+  require('~__REDWOOD__USER_STORYBOOK_PREVIEW_CONFIG')
```

Comment:
Not sure if there's a more idiomatic way to implement "require this path if it exists".